Resources/config/services.xml

```diff
@@ -43,10 +57,22 @@
             <argument type="service" id="event_dispatcher"/>
         </service>
         <!-- OpenSSL Key Loader -->
-        <service id="lexik_jwt_authentication.openssl_key_loader" class="%lexik_jwt_authentication.openssl_key_loader.class%">
-          <argument>%lexik_jwt_authentication.private_key_path%</argument>
-          <argument>%lexik_jwt_authentication.public_key_path%</argument>
-          <argument>%lexik_jwt_authentication.pass_phrase%</argument>
+        <service id="lexik_jwt_authentication.key_loader.openssl" class="%lexik_jwt_authentication.key_loader.openssl.class%">
+            <argument>%lexik_jwt_authentication.private_key_path%</argument>
+            <argument>%lexik_jwt_authentication.public_key_path%</argument>
+            <argument>%lexik_jwt_authentication.pass_phrase%</argument>
+        </service>
+        <!-- SecLib Key Loader -->
+        <service id="lexik_jwt_authentication.key_loader.seclib" class="%lexik_jwt_authentication.key_loader.seclib.class%">
+            <argument>%lexik_jwt_authentication.private_key_path%</argument>
+            <argument>%lexik_jwt_authentication.public_key_path%</argument>
+            <argument>%lexik_jwt_authentication.pass_phrase%</argument>
+        </service>
+        <!--- OpenSSL Key Loader / Old implementation (deprecated since 2.0) -->
+        <service id="lexik_jwt_authentication.openssl_key_loader" class="Lexik\Bundle\JWTAuthenticationBundle\Services\OpenSSLKeyLoader">
```

Comment:
You can officially deprecate these - which will cause the user to see a deprecation warning :) http://symfony.com/blog/new-in-symfony-2-8-deprecated-service-definitions